src/elements/content-sidebar/MetadataInstanceEditor.tsx

```diff
@@ -28,15 +28,19 @@ const MetadataInstanceEditor: React.FC<MetadataInstanceEditorProps> = ({
     setIsUnsavedChangesModalOpen,
     template,
     onCancel,
-    onUnsavedChangesModalCancel,
+    onDiscardUnsavedChanges,
 }) => {
     const handleCancel = () => {
         onCancel();
     };
 
     return (
-        <AutofillContextProvider isAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled}>
+        <AutofillContextProvider
+            isAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled}
+            fetchSuggestions={() => Promise.resolve([])}
```

Comment:
Is this going to be implemented in the future?

Reply:
this is a temporary placeholder while the feature is still being implemented in `metadata-editor`